### ISSUE TITLE
clang: error: unable to execute command: Hangup

### ISSUE THREAD

**Issue by XuanBaby** [ROCmSoftwarePlatform/tensorflow-upstream#699]
env:
rocm:2.8
gfx906
python:3.6.8
Tensorflow:1.13
task:object detection
run the tensorflow/models but there some error. one strange thing that in rocm2.8 the clang --version is 10.0.0,but program output clang version 4.0
INFO:tensorflow:global step 895: loss = 2.3266 (0.299 sec/step)
INFO:tensorflow:global step 896: loss = 2.5355 (0.340 sec/step)
INFO:tensorflow:global step 896: loss = 2.5355 (0.340 sec/step)
clang: error: unable to execute command: Hangup
clang: error: clang frontend command failed due to signal (use -v to see invocation)
clang version 4.0
Target: amdgcn-amd-amdhsa-amdgizcl
Thread model: posix
InstalledDir: /opt/rocm/opencl/bin/x86_64
clang: note: diagnostic msg: PLEASE submit a bug report to http://llvm.org/bugs/ and include the crash backtrace, preprocessed source, and associated run script.
clang: note: diagnostic msg: Error generating preprocessed source(s) - no preprocessable inputs.
MIOpen Error: /root/driver/MLOpen/src/tmp_dir.cpp:18: Can't execute cd /tmp/miopen-MIOpenConv1x1J1.cl-ce72-91cf-69c9-63a6; /opt/rocm/bin/clang-ocl  -DMLO_FILTER_STRIDE0=1 -DMLO_FILT
ER_STRIDE1=1 -DMLO_N_LCL_IN_MAPS_ONCE=8 -DBATCHSIZE=1 -DH=120 -DW=75 -DC=256 -DK=512 -DMLO_N_LCL_IN_MAPS=128 -DMLO_N_INPUTS=256 -DMLO_N_OUTPUTS=512 -DH_out=120 -DW_out=75 -DMLO_N_IN_GROUPS=2 -DMLO_CLOOP0=16 -DMLO_CLOOP2=16 -DMLO_N_LCL_OUT_MAPS=32 -DMLO_CHEAT_SHADER_COMPILER=0 -DMLopen_RUNNING=1 -DMIOPEN_USE_FP16=0 -DMIOPEN_USE_FP32=1 -DMIOPEN_USE_INT8=0 -DMIOPEN_USE_INT8x4=0 -DMIOPEN_USE_BFP16=0 -DMIOPEN_USE_INT32=0 -DMIOPEN_USE_RNE_BFLOAT16=1 -mcpu=gfx906 -Wno-everything MIOpenConv1x1J1.cl -o /tmp/miopen-MIOpenConv1x1J1.cl-ce72-91cf-69c9-63a6/MIOpenConv1x1J1.cl.o2019-11-08 07:44:57.445958: F tensorflow/stream_executor/rocm/rocm_dnn.cc:2642] Check failed: status == miopenStatusSuccess (7 vs. 0)Unable to find a suitable algorithm for doing fo
rward convolution

**Comment by sunway513** [comment]
Hi @XuanBaby , can you provide a docker image to reproduce the issue you've observed?
It looks like configuration issues, hence we won't be able to help without a proper local repro.
